src/app/presenter/ChessBoardPresenter.ts

```diff
@@ -31,4 +35,34 @@ export class ChessBoardPresenter {
     const square: Square = { column: columns[position.x - 1], row: position.y as Row };
     return `${square.column.toLowerCase()}${square.row}`;
   }
+
+  private translatePositionToSquareNotation(position: Position): Square {
+    return { column: columns[position.x - 1], row: position.y as Row };
+  }
+
+  private translateSquareToAlgebraicNotation(square: Square): string {
+    return `${square.column.toLowerCase()}${square.row}`;
+  }
+
+  private checkEvents(eventsArray: (PieceWasMoved | PieceWasCaptured)[]): void {
+    eventsArray.forEach((event) => {
+      event.eventType === 'PieceWasMoved'
+        ? this.view.movePiece(this.translateSquareToAlgebraicNotation(event.from), this.translateSquareToAlgebraicNotation(event.to))
+        : this.view.capturePiece(this.translateSquareToAlgebraicNotation(event.onSquare));
```

Comment:
Nie wiem czy to dobra rada z mojej strony, ale wyrzuciłbym to do 2 zmiennych jakoś fajnie nazwanych bo to dla mnie za długie i nie wiem co robi true a co false